### DISCUSSION TITLE
about expanding spatial structure

### DISCUSSION THREAD

**Discussion by Achineseman** [Open-Systems-Pharmacology/Forum#960] [Questions & Problems] [answered]
I expanded the structure according to the tutorial, but found that the cardiac output formula was different from the tutorial. Is it because of different software versions? Does this formula contain the new structure by default? If not,do I need to change this formula?
 Software Version 9.1
<img width="960" alt="屏幕截图 2021-08-30 190000" src="https://user-images.githubusercontent.com/81560924/131339856-04201b97-a734-44f4-9968-98041b8cae35.png">
<img width="685" alt="屏幕截图 2021-08-30 190115" src="https://user-images.githubusercontent.com/81560924/131339891-e236c68a-cbf8-406f-a34e-665705b0db99.png">
<img width="725" alt="屏幕截图 2021-08-30 202830" src="https://user-images.githubusercontent.com/81560924/131339902-773fb2d3-94e7-409b-9beb-250421cf0085.png">

**Comment by StephanSchaller** [comment, marked as answer]
Hi @Achineseman ,
yes, this was changed to a sum formula so you no longer need to update this formula as it will automatically add the "Q" (Blood Flow) from your new organ to the total (lung) blood flow.

**Reply by Achineseman** [reply]
Thank you for your help
